src/DbInterface.cpp

```diff
@@ -848,6 +895,8 @@ void DbInterface::processMuxLinkmgrConfigNotifiction(std::deque<swss::KeyOpField
                         mMuxManagerPtr->setSuspendTimeout_msec(boost::lexical_cast<uint32_t> (v));
                     } else if (f == "use_well_known_mac") {
                         mMuxManagerPtr->setUseWellKnownMacActiveActive(v == "enable");
+                    } else if (f == "src_mac") {
+                        mMuxManagerPtr->processSrcMac(v == "ToRMac");
```

Comment:
The default could be vlan mac?

Reply:
The default is vlan mac. This is to fall back.